nmigen_boards/versa_ecp5.py

```diff
@@ -171,6 +166,31 @@ def toolchain_program(self, products, name):
             ])
 
 
+class VersaECP5Platform(_VersaECP5PlatformBase):
+    resources = (
+        _VersaECP5PlatformBase._resources +
+        [
+            *SPIFlashResources(0,
+                cs="R2", clk="U3", miso="V2", mosi="W2", wp="Y2", hold="W1",
+                attrs=Attrs(IO_STANDARD="LVCMOS33")
+            )
+        ]
+    )
+
+
+class VersaECP5Platform_USRMCLK(_VersaECP5PlatformBase):
```

Comment:
Probably better to add a parameter to ``__init__`` instead of having two separate classes. This is what we did in other places (e.g. Migen), having one class for each combination of options does not scale.